src/poprox_storage/repositories/accounts.py

```diff
@@ -106,6 +107,24 @@ def store_new_account(self, email: str, source: str, subsource: str = None) -> A
             account_id=row.account_id, email=row.email, status=row.status, source=source, subsource=subsource
         )
 
+
+######################### storing & fetching the zip code #########################
+    
+    def store_zip_code(self, account_id: UUID, zip5: str) -> bool:
```

Comment:
given that we're using specifically zip5 vs. zip3 (and zip+4 logically does exist) should we update this to `store_zip5`?

or similar?

~Kluver

Reply:
Changed the naming.